### PR TITLE
Refactor Worldrender +Portrait support + rescaling UI

### PR DESCRIPTION
**Pull request by Jurgenvs**
The worldrenderer has been refactored slightly to fit the new requirements. These changes include the portrait support for the application as well as the 'always sharp' UI. Elements are redrawn whenever a resize happens..

### CODE REVIEW

File: core/src/cg/group4/view/screen_mechanics/WorldRenderer.java [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package cg.group4.view.screen_mechanics;
 
-import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.Screen;
-import com.badlogic.gdx.audio.Music;
+import com.badlogic.gdx.*;
```
Comment (sirpolomotro) [inline core/src/cg/group4/view/screen_mechanics/WorldRenderer.java]: .* in imports is bad practice

**Comment by sirpolomotro** [comment]
If the change the rotation of the screen during the TestStrollEvent, the accelerometer wont change what he think is up down left and right. So this is a bug it should reset the directions when the rotation of the phone is changed or it should be programmed more dynamically.

**Comment by sirpolomotro** [comment]
Text is sometime out of the bounds of the screen.

**Comment by Jurgenvs** [comment]
Made issues from the feedback you gave. The accelerometer is something out of the scope for this branch. The text issue is something that needs to be looked at later, but is not of highest priority now.

**Pull request merged**